testapps/build_pipeline/app.yaml

```diff
@@ -0,0 +1,8 @@
+runtime: php
+env: flex
+
+runtime_config:
+  document_root: web
+
+env_variables:
+  WHITELIST_FUNCTIONS: phpinfo,exec
```

Comment:
I think we should move it here once we switch to the builders. All the other examples use env variables for now.